Lib/platform.py

```diff
@@ -110,7 +110,7 @@
 
 """
 
-__version__ = '1.0.9'
+__version__ = '1.0.10'
```

Comment:
I don't think this should be updated?

```suggestion
__version__ = '1.0.9'
```

It was updated a couple of months ago in https://github.com/python/cpython/pull/122547, but the previous change was 10 years ago (https://github.com/python/cpython/commit/b9f4feab1b9c9ffa8ea29af3d82bc536f9f3005a), and there have been lots of updates in the past decade that didn't change it:

https://github.com/python/cpython/commits/main/Lib/platform.py

Reply:
Is the module __version__ attribute really still needed?  I thought that these were obsolete.  If kept, is the use defined anywhere in a manner that would say when to bump it?

Reply:
As I understand it just wasn't kept properly. Each change should increase the version (path, minor, major as needed).

Reply:
btw we've decided to deprecate these module `__version__`s: https://github.com/python/cpython/issues/76007 and have done `platform` in https://github.com/python/cpython/pull/138675.